README.md

```diff
@@ -261,6 +200,44 @@ Array<{
     address: string,
     amount: string,
   }>,
+  withdrawals: Array<{ address: string,
+    amount: string
+  }>,
+  certificates: Array<{|
```

Comment:
I think the `cert_index` field is missing in this.

In the Yoroi codebase I added it like so

```
export type RemoteCertificate = {|
  cert_index: number,
  ...({|
    +type: typeof ShelleyCertificateTypes.StakeRegistration,
    ...RemoteStakeRegistrationCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.StakeDeregistration,
    ...RemoteStakeDeregistrationCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.StakeDelegation,
    ...RemoteStakeDelegationCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.PoolRegistration,
    ...RemotePoolRegistrationCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.PoolRetirement,
    ...RemotePoolRetirementCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.GenesisKeyDelegation,
    ...RemoteGenesisKeyDelegationCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.MoveInstantaneousRewardsCert,
    ...RemoteMoveInstantaneousRewardsCert,
  |})
|};
```

```
export type RemoteCertificate = {|
  +type: typeof ShelleyCertificateTypes.StakeRegistration,
  cert_index: number,
  ...RemoteStakeRegistrationCert,
  ...({|
|} | {|
    +type: typeof ShelleyCertificateTypes.StakeRegistration,
  +type: typeof ShelleyCertificateTypes.StakeDeregistration,
    ...RemoteStakeRegistrationCert,
  ...RemoteStakeDeregistrationCert,
  |} | {|
|} | {|
    +type: typeof ShelleyCertificateTypes.StakeDeregistration,
  +type: typeof ShelleyCertificateTypes.StakeDelegation,
    ...RemoteStakeDeregistrationCert,
  ...RemoteStakeDelegationCert,
  |} | {|
|} | {|
    +type: typeof ShelleyCertificateTypes.StakeDelegation,
  +type: typeof ShelleyCertificateTypes.PoolRegistration,
    ...RemoteStakeDelegationCert,
  ...RemotePoolRegistrationCert,
  |} | {|
|} | {|
    +type: typeof ShelleyCertificateTypes.PoolRegistration,
  +type: typeof ShelleyCertificateTypes.PoolRetirement,
    ...RemotePoolRegistrationCert,
  ...RemotePoolRetirementCert,
  |} | {|
|} | {|
    +type: typeof ShelleyCertificateTypes.PoolRetirement,
  +type: typeof ShelleyCertificateTypes.GenesisKeyDelegation,
    ...RemotePoolRetirementCert,
  ...RemoteGenesisKeyDelegationCert,
  |} | {|
|} | {|
    +type: typeof ShelleyCertificateTypes.GenesisKeyDelegation,
  +type: typeof ShelleyCertificateTypes.MoveInstantaneousRewardsCert,
    ...RemoteGenesisKeyDelegationCert,
  ...RemoteMoveInstantaneousRewardsCert,
  |} | {|
    +type: typeof ShelleyCertificateTypes.MoveInstantaneousRewardsCert,
    ...RemoteMoveInstantaneousRewardsCert,
  |})
|};
|};